www/app/login/login.controller.js

```diff
@@ -0,0 +1,51 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('budgetrentacar.login')
+    .controller('LoginController', ['$scope', '$state', '$ionicPopup', '$localStorage', 'AuthService', function($scope, $state, $ionicPopup, $localStorage, AuthService) {
+      var ctrlScope = $scope;
+      ctrlScope.user = { };
+
+      ctrlScope.createAccount = function(){
+        $ionicPopup.alert({
+          tittle: 'Budget Rent a Car',
+          template: 'Not available in this prototype'  
+        }).then(function(res){
+          console.log('Show alert')
+        });
+      };
+
+      ctrlScope.login = function(){
+        console.log("login submit : " + ctrlScope.user);
```

Comment:
`console.log`s are cool when you're developing and working on your local box, you know, to find problems and stuff... but get rid of them before pushing to git.

NO CONSOLE LOGS ARE ALLOWED IN PRODUCTION.